src/librustdoc/json/conversions.rs

```diff
@@ -222,7 +219,11 @@ impl From<clean::Union> for Struct {
             struct_type: struct_type.into(),
             generics: generics.into(),
             fields_stripped,
-            fields: fields.into_iter().map(|i| i.def_id.into()).collect(),
+            fields: fields
+                .into_iter()
+                .filter(|x| !x.is_stripped())
+                .map(|i| i.def_id.into())
+                .collect(),
```

Comment:
Why not use `ids()` here?

Reply:
Changed that one manualy, realized this should probably be a function, searched for the old pattern and missed this.